modules/log-collector-rsyslog-server.adoc

```diff
@@ -36,13 +39,42 @@ spec:
   pipelines: # <4>
     - name: syslog-pipeline
       inputRefs:
-        - syslog-receiver
+        - infrastructure
```

Comment:
This must match the input name.  We can't use infrastructure here because it is considered a reserved word

Reply:
This info came from #3 on https://issues.redhat.com/browse/OBSDOCS-890 - can you please check this with @r2d2rnd and confirm? It seems like he had issues getting it to work without this setup